context/app/static/js/shared-styles/Links/index.js

```diff
@@ -10,4 +11,10 @@ const LightBlueLink = styled(LinkWithoutUnderline)`
   color: ${(props) => props.theme.palette.info.main};
 `;
 
-export { LightBlueLink };
+const StyledOpenInNewRoundedIcon = styled(OpenInNewRoundedIcon)`
```

Comment:
Is this a change necessary for all instances of `OutboundLink`?

Reply:
This seems to have been moved here from the `style.js` in the same directory. I think that this and the `LightBlueLink` should both be in `style.js` instead of `index.js` in order to prevent `OutboundLinkButton` from importing from the index file of its own directory (which can cause a dependency loop)